src/UICrafter.Client/Components/AppViewButtons.razor

```diff
@@ -0,0 +1,59 @@
+@using static UICrafter.Client.Pages.UICrafterPage
```

Comment:
Is this necessary? If it is something that is used on multiple pages it can be added to _Imports.razor that will make them all available.

Reply:
Midlertidigt mens UIComponent models er i UICrafterPage koden